contrib/Dockerfile

```diff
@@ -0,0 +1,32 @@
+# Usage
+#
+#   cd contrib
```

Comment:
Is there a reason why you put the Dockerfile in `contrib` and not in the root directory?

Reply:
It makes more sense to me, because it this Dockerfile doesn't use anything from the repository. Both apps are installed with `pip3 install pyatv`  and `npm install -g @sebbo2002/pyatv-mqtt-bridge` from external sources.

Usually you would add the contents from this repository in a Dockerfile (e.g. like [this](https://github.com/dersimn/spotify-archiver/blob/79a9dbf20178fddb794afc632064f8762b16eb48/Dockerfile#L3)). 

I'd say this Dockerfile is more like a hack after all. If you like Docker and also want to provide an image, maybe this can help: https://github.com/dersimn/HelloARM#github-workflow-recommended